src/main/java/pl/tomaszdziurko/jvm_bloggers/view/login/LoginPage.java

```diff
@@ -65,11 +75,11 @@ private void tryToLoginUser(String clientAddress) {
 
         CustomFeedbackPanel feedbackPanel = new CustomFeedbackPanel("feedbackPanel");
         loginForm.add(feedbackPanel);
-        RequiredTextField<String> loginField = new RequiredTextField<>(LOGIN_FIELD_ID);
+        RequiredTextField loginField = new RequiredTextField(LOGIN_FIELD_ID);
```

Comment:
You are right :+1: We have to define what do we mean by so called "The Boy Scout Rule". In short, change scope  matters. By default, when we are going to change code in a method, we are blessed to clean code within it whereas other methods should stay as they are. This and other agreed rules should be described in contributing guideline.